src/utils/boilerplate-cli.ts

```diff
@@ -38,18 +38,19 @@ class BoilerplateCli {
         if (!command.code) {
             shell.cd(this.dirName);
             this.insertNameProject();
-            shell.exec('rm -rf .git');
+            if (this.boilerplate === 'juggernaut') {
+              shell.exec('rm -rf .git .github netlify.toml', {async: true});
```

Comment:
You guys can always use `rimraf` or change the `shell.exec` to `rm -rf .git* netlify.toml`